lighthouse-core/report/html/report-styles.css

```diff
@@ -1010,16 +1011,21 @@
 
 .lh-warnings--toplevel {
   --item-margin: calc(var(--header-line-height) / 4);
-  color: var(--report-text-color-secondary);
+  color: var(--report-text-color);  
```

Comment:
can you make a `--toplevel-warning-text-color` variable, initialized to `var(--report-text-color)`, and vary it to `var(--color-orange-700)` in dark mode? Trying to solve these things:

1. color contrast in light mode for this text fails the axe audit. Let's ignore the color given in the design for that and just use black.
2. we try not to use colors directly in the css, but instead alias to a component variable. see the "naming" comment at the top of the file

Reply:
I may have misunderstood this so apologies if this needs corrected. 

I believe it was the text "There were issues affecting this run of Lighthouse:" that was causing the contrast issue, the text color for this is controlled by the .lh-warnings__msg class so I have created another component variable for this text color along with the --toplevel-warning-text-color requested. 

I think I've followed the correct conventions for this but I'm unsure if the result is exactly what you were looking for. 

![image](https://user-images.githubusercontent.com/13604740/81476986-d9d37c00-920c-11ea-98ce-1807f8f36de6.png)

![image](https://user-images.githubusercontent.com/13604740/81476997-e35ce400-920c-11ea-9c8c-6182282fafc6.png)